### PR TITLE
HP-1585: add link and filter by bill

### PR DESCRIPTION
**Pull request by bladeroot**

### CODE REVIEW

File: src/menus/BillActionsMenu.php [hunk 1]
```diff
@@ -54,6 +54,15 @@ public function items()
                     'data-pjax' => 0,
                 ],
             ],
+            [
+                'label' => Yii::t('hipanel:finance', 'Charges'),
+                'icon' => 'fa-user',
+                'url' => ['/finance/charge/index', 'ChargeSearch' => ['bill_id_in' => $this->model->id]],
```
Comment (SilverFire) [inline src/menus/BillActionsMenu.php:60]:
```suggestion
                'url' => Url::toSearch('charge', ['bill_id_in' => $this->model->id]),
```

File: src/menus/BillActionsMenu.php [hunk 2]
```diff
@@ -54,6 +54,15 @@ public function items()
                     'data-pjax' => 0,
                 ],
             ],
+            [
+                'label' => Yii::t('hipanel:finance', 'Charges'),
+                'icon' => 'fa-user',
```
Comment (SilverFire) [inline src/menus/BillActionsMenu.php:59]: @tafid help choosing better icon, please

File: src/views/charge/_search.php [hunk 3]
```diff
@@ -41,6 +41,10 @@
     ]) ?>
 </div>
 
+<div class="col-md-4 col-sm-6 col-xs-12">
+    <?= $search->field('bill_id_in') ?>
+</div>
+
```
Comment (SilverFire) [inline src/views/charge/_search.php:47]:
```suggestion
```

It's counter-intuitive to search by a list of IDs
What is this filed for?

**Review by SilverFire — CHANGES_REQUESTED** [review]

File: src/views/charge/_search.php (hunk 3 above)
Comment (bladeroot) [reply src/views/charge/_search.php:47]: Fist requirements of task )

Comment (SilverFire) [reply src/views/charge/_search.php:47]: @hiqsol please explain

Comment (hiqsol) [reply src/views/charge/_search.php:47]: it's written in the issue - to allow export charges of the given bill(s)

File: src/menus/BillActionsMenu.php (hunk 2 above)
Comment (hiqsol) [reply src/menus/BillActionsMenu.php:59]: eg. `fa-list` or `fa-th-list`

**Review by hiqsol — APPROVED** [review]

File: src/views/charge/_search.php (hunk 3 above)
Comment (SilverFire) [reply src/views/charge/_search.php:47]: It's not a user-friendly UI.

How about adding a bulk action on the bills index page? Then you can choose multiple bills with checkboxes, click on "view charges" button and get redirected to the charge index page with preset filters?

File: src/menus/BillActionsMenu.php (hunk 2 above)
Comment (SilverFire) [inline src/menus/BillActionsMenu.php:59]:
```suggestion
                'icon' => 'fa-th-list',
```

**Review by SilverFire — APPROVED** [review]

**Pull request merged**